### PR TITLE
Add LIP "Define state transitions of Reward module"

### PR DESCRIPTION
**Pull request by IkerAlus**

### CODE REVIEW

File: proposals/lip-define_state_transitions_of_reward_module.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+`blockHeader`: A block header.
+
+##### Returns
+
+The reward for the block as a 64-bit unsigned integer.
+
+##### Execution
+
+```
```
Comment (ricott1) [inline proposals/lip-define_state_transitions_of_reward_module.md:95]:
```suggestion
```python
```

File: proposals/lip-define_state_transitions_of_reward_module.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+The default reward of the block as a 64-bit unsigned integer.
+
+### Block Processing
+
+#### After Block Execution
+
+After a block `b` is executed, the `mint` function exposed by the [token module][tokenLIP] to assign the reward to the block generator as:
+
+```
```
Comment (ricott1) [inline proposals/lip-define_state_transitions_of_reward_module.md:130]:
```suggestion
```python
```

File: proposals/lip-define_state_transitions_of_reward_module.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+* After applying the block, a certain quantity of the native token of the blockchain is minted and assigned to the block generator. The exact amount assigned to the block generator, i.e., the reward, is calculated depending of the rules of the [random module][randomLIP] and the [BFT module][BFTLIP].
+
+## Specification
+
+In this section, we specify the protocol logic in the lifecycle of a block injected by the reward module as well as the functions that can be called from off-chain services.
+This module does not contain commands, and has an empty state.
+It depends on the token, BFT, and random modules.
+
+### Constants
+
```
Comment (ricott1) [inline proposals/lip-define_state_transitions_of_reward_module.md:40]: You don't have the description column in the table

**Review by ricott1 — CHANGES_REQUESTED** [review]
Missing description in the table, code snippets

File: proposals/lip-define_state_transitions_of_reward_module.md (hunk 3 above)
Comment (IkerAlus) [reply proposals/lip-define_state_transitions_of_reward_module.md:40]: is it always a must? specially for such small self explanatory table.

File: proposals/lip-define_state_transitions_of_reward_module.md (hunk 1 above)
Comment (IkerAlus) [reply proposals/lip-define_state_transitions_of_reward_module.md]: added

File: proposals/lip-define_state_transitions_of_reward_module.md (hunk 2 above)
Comment (IkerAlus) [reply proposals/lip-define_state_transitions_of_reward_module.md]: added

**Review by janhack — APPROVED** [review]
Looks all good, nice work!

**Review by ricott1 — APPROVED** [review]
bravo Iker!

**Review by mehmetegemen — APPROVED** [review]
Good job Iker!

**Review by ManuGowda — APPROVED** [review]

**Review by karmacoma — APPROVED** [review]

**Pull request merged**